src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade-form/upgrade-start-modal.component.ts

```diff
@@ -42,8 +42,12 @@ export class UpgradeStartModalComponent implements OnInit {
     this.upgradeForm = new CdFormGroup({
       availableVersions: new FormControl(null, [Validators.required]),
       useImage: new FormControl(false),
-      customImageName: new FormControl(null)
+      customImageName: new FormControl(null, [Validators.required]])
```

Comment:
@avanthakkar I think this will because on the regular start-upgrade form, we are hiding this initially. So if the user just casually selects a version and proceeds to upgrade that won't happen. That's why I told to add the required validator in the newly added `if (versions)` block